tools/wasi/wasi.cpp

```diff
@@ -132,7 +132,7 @@ bool run(int argc, const char** argv)
         nullptr,  // NOTE: no special allocator
     };
 
-    uvwasi_errno_t err = uvwasi_init(&state, &options);
+    const uvwasi_errno_t err = uvwasi_init(&state, &options);
```

Comment:
Why is this const stuff so important on this?

Reply:
The `const` should be the default.

Reply:
Should protest C++ to follow Rust with making const-by-default and have a mutable specifier on others :)

Reply:
We both know this is not possible.